domain/snyk/issues.go

```diff
@@ -101,6 +101,7 @@ type CodeIssueData struct {
 	HasAIFix           bool               `json:"hasAIFix"`
 	DataFlow           []DataFlowElement  `json:"dataFlow,omitempty"`
 	Details            string             `json:"details"`
+	LessonUrl          string             `json:"url"`
```

Comment:
suggestion: this could even be added one level up, as we have learn lessons for all products, not only code. WDYT?

Reply:
It makes sense since `OssIssueData` implements it in its own way, so we could unify them by moving `LessonUrl` to the Issue struct as you suggest.